cni/ipam/ipam_test.go

```diff
@@ -4,17 +4,20 @@
 package ipam
 
 import (
+	"encoding/json"
 	"fmt"
+	"github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/platform"
+	cniSkel "github.com/containernetworking/cni/pkg/skel"
+	cniTypesCurr "github.com/containernetworking/cni/pkg/types/current"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
are these 3rd party packages signed by microsoft legal (lca)

Reply:
I am not sure about this, but we have these two packages used in repos like aks-engine already.
https://github.com/search?q=org%3AAzure+github.com%2Fonsi%2Fginkgo&type=Code